inception_v3.py

```diff
@@ -297,13 +297,13 @@ def InceptionV3(images,
     # Global average pooling. Assumes channels_last
     x = tf.reduce_mean(x, axis=[1, 2])
     logits = layers.dense(x, classes, activation=None, name='logits')
-    predictions = tf.nn.softmax(logits, name='predictions')
+    predictions = tf.argmax(logits, axis=1, output_type=tf.int32, name='predictions')
 
-    loss = tf.losses.softmax_cross_entropy(labels, logits)
-    train_step = (tf.train.RMSPropOptimizer(learning_rate=0.001)
+    loss = tf.losses.sparse_softmax_cross_entropy(labels, logits)
+    train_step = (tf.train.GradientDescentOptimizer(learning_rate=0.1)
```

Comment:
The original paper uses the RMSProp optimizer. I set the learning rate was set as the Keras default though, that may need some tuning

```suggestion
    train_step = (tf.train.RMSPropOptimizer(learning_rate=0.001)
```

Reply:
That was an attempt to debug given that RMS wasn't converging--we can set it back at any point (although the issues appear to persist regardless of the choice of optimizer)